### PR TITLE
docs: update docs wrt oauth scope changes

### LINKED ISSUE #551: Documentation and Spec needs updating for scope changes
Existing documentation (tutorials, respec, etc.) need to be updated to reflect the changes made to scope requirements in PR #545

### PR DESCRIPTION
**Pull request by brownoxford**
This PR updates documentation (openapi, respec, tutorials, etc.) based on recent changes to how we handle OAuth2 scopes per #457

Fixes #551

### CODE REVIEW

**Review by OR13 — APPROVED** [review]

File: docs/tutorials/authentication/README.md [hunk 1]
```diff
@@ -38,6 +38,10 @@ Add the following variable configuration to the "Authentication Tutorial" Postma
 <dl>
   <dt><code>CLIENT_ID</code></dt>
   <dd>The client ID obtained from your OAuth service provider.</dd>
+  <dt><code>CLIENT_SCOPE</code></dt>
+  <dd>
+    The names of the scopes to request from your OAuth service provider. If your OAuth service provider requires that you name the specific scopes that should be included in the auth token, you should provide a value for this variable.
```
[the remark this answers is not in the document]
Comment (OR13) [reply docs/tutorials/authentication/README.md:43]: this should be further improved.

The key thing here is to emphasize the following:

1. avoid asking for scopes you don't need (least priv)
2. support fine grained scopes when possible
3. avoid extending OIDC or OAuth in custom ways, as this destroys the ability to easily onboard.

File: docs/tutorials/traceable-presentation-workflow/README.md [hunk 2]
```diff
@@ -24,6 +24,7 @@ After the collection has been imported, you should see a new collection in your
 4. Add the following variables to the environment, replacing the placeholders with your own values. These should reflect the Machine to Machine OAuth Application configured on the Verifier's platform, used for the Holder to make the Traceable Presentation which includes Workflow details:
    - `API_BASE_URL`: Verifier platform's base URL. 
    - `CLIENT_ID`: The client ID for the M2M application, to be used by the Holder.
+   - `CLIENT_SCOPE`: The names of the scopes to request from your OAuth service provider. If your OAuth service provider requires that you name the specific scopes that should be included in the auth token, you should provide a value for this variable.
```
Comment (OR13) [inline docs/tutorials/traceable-presentation-workflow/README.md:27]:
```suggestion
   - `CLIENT_SCOPE`: The names of the scopes to request from your OAuth service provider.
```

**Comment by mkhraisha** [comment]
Please address CR, and merge offline

File: docs/tutorials/traceable-presentation-workflow/README.md (hunk 2 above)
Comment (TallTed) [reply docs/tutorials/traceable-presentation-workflow/README.md:27]:
```suggestion
   - `CLIENT_SCOPE`: The names of the scopes to request from your OAuth service provider. Optional. If your OAuth service provider requires that you name the specific scopes that should be included in the auth token (as Azure does), you should provide a value for this variable.
```

**Comment by mkhraisha** [comment]
@brownoxford any updates to address the CR?

**Comment by mkhraisha** [comment]
@brownoxford closing this PR in one week if there is no movement, unless you respond otherwise

**Comment by nissimsan** [comment]
@brownoxford , we'll close this. Pls reopen if we need to progress this.

**Pull request closed without merging**